### ISSUE TITLE
Bibtex issue

### ISSUE THREAD

**Issue by vrossum** [carlobaldassi/LyX_PNAS#1]
Hi Carlo
As soon as you specify the bibliography style as pnas-new (or anything else, I think)
lyx complains during compilation.

Mark

**Comment by carlobaldassi** [comment]
Yes, I'm not sure why. I think that their style specifies it already. However, if you just leave it unspecified, it works and uses the correct style, I think.
